metadata-ingestion/transformers.md

```diff
@@ -121,6 +117,8 @@ transformers:
         - /data_warehouse/DATASET_PARTS/
 ```
 This will add 2 browse paths like `/mysql/marketing_db/sales/orders` and `/data_warehouse/sales/orders` for a table `sales.orders` in `mysql` database instance.
+
+Note that whatever browse paths you send via this will overwrite the owners present in the UI.
```

Comment:
not sure what you mean by this. is this a typo on `owners` ?

Reply:
fixed typo